src/utils/filter-utils.js

```diff
@@ -413,7 +414,13 @@ export function getFilterFunction(field, dataId, filter, layers) {
     case FILTER_TYPES.range:
       return data => isInRange(valueAccessor(data), filter.value);
     case FILTER_TYPES.multiSelect:
-      return data => filter.value.includes(valueAccessor(data));
+      return data => {
+        const value = valueAccessor(data);
+        if (Array.isArray(value)) {
```

Comment:
To make this function more efficient, we should do the checking out side the `data => boolean` function
```js
case FILTER_TYPES.multiSelect:
return feld.type === ALL_FILED_TYPES.array ? data => {
  const value = valueAccessor(data);
  return Array.isArray(value) && value.some(v => filter.value.includes(v));
} : data => filter.value.includes(valueAccessor(data));
```

Reply:
Indeed it's better, I'll update this one.